### PR TITLE
docs: add new example with AI & R2

### PR DESCRIPTION
**Pull request by atinux**
### Summary

Repo: https://github.com/atinux/flux-ai-image-generator

### Screenshots (optional)

See in videos: https://x.com/Atinux/status/1843256114479284360

### Documentation checklist

- [x] The [documentation style guide](https://developers.cloudflare.com/style-guide/) has been adhered to.

### CODE REVIEW

File: src/content/apps/index.yaml [hunk 1]
```diff
@@ -333,3 +333,11 @@ entries:
     languages: [TypeScript]
     cloudflare: true
     updated: 2024-09-12
+  - link: https://github.com/atinux/flux-ai-image-generator
+    name: Flux Schnell Image Generator
+    description: An application to generate images with AI using Flux-1 Schnell and storing them to Cloudflare R2.
+    tags: [Nuxt]
+    products: [Pages, Workers AI, R2]
+    languages: [TypeScript]
+    cloudflare: true
```
Comment (dario-piotrowicz) [inline src/content/apps/index.yaml]:
```suggestion
    cloudflare: false
```

**Review by dario-piotrowicz — APPROVED** [review]
Nice! 😄 🚀

File: src/content/apps/index.yaml [hunk 2]
```diff
@@ -333,3 +333,11 @@ entries:
     languages: [TypeScript]
     cloudflare: true
     updated: 2024-09-12
+  - link: https://github.com/atinux/flux-ai-image-generator
+    name: Flux Schnell Image Generator
+    description: An application to generate images with AI using Flux-1 Schnell and storing them to Cloudflare R2.
```
Comment (dcpena) [inline src/content/apps/index.yaml]:
```suggestion
    description: An application to generate images with AI using Flux-1 Schnell and store them in Cloudflare R2.
```

Comment (dcpena) [reply src/content/apps/index.yaml]: One very small edit!

Comment (atinux) [reply src/content/apps/index.yaml]: Thank you @dcpena !

**Review by dcpena — APPROVED** [review]

**Pull request merged**